### DISCUSSION TITLE
whether the error message is misleading

### DISCUSSION THREAD

**Discussion by shaobo322** [apache/apisix#5856] [General]
Hi there, as show in the picture below, when we create a etcd cli, if failed, we reture an error message `failed to start a etcd instance: ......`,  but actually we do not try to start an etcd instance, what we do is just create a long connect to an already existed etcd instance
![image](https://user-images.githubusercontent.com/37969177/146704139-fc1c8609-dfe9-4a7b-b026-45a45e368e12.png)

**Comment by tzssangglass** [comment]
we can change `instance` to `connection`?

**Reply by tzssangglass** [reply]
`failed to create a etcd connection`
